AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -165,16 +165,12 @@ - (void)_layoutNodes:(NSArray<ASCellNode *> *)nodes fromContexts:(NSArray<ASInde
     return;
   }
   
-  // For any given layout pass that occurs, this method will be called at least twice, once on the main thread and
-  // the background, to result in complete coverage of both loaded and unloaded nodes
-  BOOL isMainThread = ASDisplayNodeThreadIsMain();
+  // Layout node on whatever thread we are on. We handle the trampoline to the main thread in case the node is
```

Comment:
Don't you mean ASDisplayNode (rather than We = ASDataController) handles the trampoline?